app/main.py

```diff
@@ -11,4 +13,5 @@ def convert_to_human(
         return 0
     if animal_age < first_year + second_year:
         return 1
-    return 2 + (animal_age - first_year - second_year) // each_year
+    remaining_years = animal_age - first_year - second_year
```

Comment:
There is an issue with the calculation of `remaining_years`. The subtraction operation should be fully enclosed in parentheses to ensure correct precedence. Currently, `second_year` is divided by `each_year` before being subtracted, which is not the intended logic. It should be: `remaining_years = animal_age - (first_year + second_year)`.